datarepo-clienttests/build.gradle

```diff
@@ -40,7 +40,7 @@ dependencies {
         swaggerAnnotations = "1.5.22"
         jersey = "2.30.1"
 
-        datarepoClient = "1.0.39-SNAPSHOT"
+        datarepoClient = "1.0.104-SNAPSHOT"
```

Comment:
Would it make more sense inheriting this version from jade-data-repo by making this a subproject (like how the datarepo-client is now?)

Reply:
Automatically updating the version sounds good. I'm not sure what else making this a sub-project would affect.

I'll add a ticket to investigate, and probably check-in with you after I read up some. Thanks for the suggestion!

Reply:
Do we want TestRunner to remain TDR-only? Or will we factor it out so other teams can use it?